### DISCUSSION TITLE
Pegar posição do último foco cursor no Componente Rich Text

### DISCUSSION THREAD

**Discussion by h2danilo** [po-ui/po-angular#1648] [Tire suas dúvidas] [answered]
Estou precisando pegar a posição do último foco do cursor no component Rich Text para adicionar algo no texto.

Ex: Conforme imagem abaixo, gostaria de posicionar cursor no componente RichText na posição que desejar, clicar na lista de botões ao lado (em azul - 2) e assim adicionar o texto do botão selecionado no componente RichText.
![image](https://user-images.githubusercontent.com/38380833/221954652-efea71ac-0635-424d-b73a-b9577319c810.png)

Alguém tem alguma sugestão ou poderia me ajudar?

**Comment by h2danilo** [comment]
Alguém pode me ajudar?

**Reply by alinelariguet** [reply, marked as answer]
Boa tarde @h2danilo hoje não é possível ter acesso à posição do mouse, você pode abrir uma issue de melhoria ou então contribuir com esta funcionalidade.
Att.
